src/ActionList/Header.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import styled, {css} from 'styled-components'
+import {get} from '../constants'
+
+export interface HeaderProps extends React.ComponentPropsWithoutRef<'div'> {
+  variant: 'subtle' | 'filled'
+  title: string
+  auxiliaryText?: string
+}
+
+const StyledHeader = styled.div<{variant: HeaderProps['variant']}>`
+  padding: calc((${get('space.3')} - ${get('space.1')}) / 2) ${get('space.2')};
+  font-size: ${get('fontSizes.0')};
+  font-weight: ${get('fontWeights.bold')};
+  color: ${get('colors.text.secondary')};
+
+  ${({variant}) =>
+    variant === 'filled' &&
+    css`
+      background: ${get('colors.bg.gray')};
```

Comment:
To make this color mode compatible, we should be using functional color variables instead of presentational variables. 

@VanAnderson wrote a helpful [migration guide](https://github.com/primer/components/pull/1115/files#diff-c967cab7c97054be52f101d2dd4c8caa124c4bf9f7415317070bcd0d1e069464).

```suggestion
      background: ${get('colors.bg.tertiary')};
```

Reply:
Fixed in https://github.com/primer/components/pull/1078/commits/1008dce035419634f413e4edc99191dcfc113030!